web/src/modules/layout/containers/__tests__/__snapshots__/LocationHeader.spec.js.snap

```diff
@@ -51,7 +51,22 @@ Array [
 ]
 `;
 
-exports[`LocationHeader NavigationItems should be empty, if offers and news are both disabled 1`] = `Array []`;
+exports[`LocationHeader NavigationItems should be empty, if offers and news are both disabled 1`] = `
```

Comment:
Something is not right here...

Reply:
I think thats because of POIs, let me have a look.

Reply:
Added a todo, a flag has to be added to disable pois depending on the flag from the cms. This will be done in https://issues.tuerantuer.org/browse/IGAPP-115.